### PR TITLE
Removed iframe-height.js and its implementations

### LINKED ISSUE #1059: Remove dead iframe height code
## Problem

Openverse is now using a hardcoded height for the iframe. We can remove any iframe code related to sending the height of the window to the parent site.

To my knowledge that is this file:

https://github.com/WordPress/openverse-frontend/blob/912f1c4a9715619120287cd45ac1ea791b9c529e/src/mixins/iframe-height.js#L4

And anywhere it is implemented.

### PR DESCRIPTION
**Pull request by BarokDG**
## Fixes

Fixes #1059 by @zackkrida

## Description

I deleted iframe-height.js inside src/mixins/ and removed all imports of that file inside src/layouts/blank.vue, src/layouts/default.vue, and src/pages/audio/_id.vue. I also removed a comment explaining the use of a resize event but undid it in a second commit. 

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by BarokDG** [comment]
I can't figure out how to add labels. Can anyone tell me?

**Comment by sarayourfriend** [comment]
This is great @BarokDG thanks for the contribution and welcome to Openverse! :tada:

I think we can keep the change you made originally to remove the `resize`, what was the reason for reverting that?

**Comment by BarokDG** [comment]
> This is great @BarokDG thanks for the contribution and welcome to Openverse! 🎉
> 
> I think we can keep the change you made originally to remove the `resize`, what was the reason for reverting that?

I couldn't see where the 'resize' value was used and there is another reference to 'resize' at the top of the comment. I wanted confirmation before I changed it.

**Comment by sarayourfriend** [comment]
> Should I remove both references?

Please do, thanks!

**Comment by BarokDG** [comment]
Done! I also removed a switch case for the 'resize' event in iframe-test.html. 

Anything I can do about the "CI/Check translation files" check?

**Comment by krysal** [comment]
@BarokDG You only need to run `pnpm i18n:generate-pot` and commit the updated .pot file to pass that check.

File: src/middleware/middleware.js [hunk 1]
```diff
@@ -12,8 +12,7 @@ import { useNavStore } from '~/stores/nav'
  *
  * Messages sent to the outer window have the following format:
  * {type: <event type>, value: <event value>}.
- * Currently, two event types are used:
- * - `resize` sends the height of the window (see `src/mixins/iframe-height.js`)
+ * Currently, one event types is used:
```
Comment (zackkrida) [inline src/middleware/middleware.js:15]:
```suggestion
 * Currently, one event type is used:
```

Small typo fix, now that there's only one item.

**Comment by BarokDG** [comment]
I followed the instructions but the "CI=1 pnpm i18n:generate-pot" command is returning a success message. I decided to try and just commit an updated openverse.pot file but the check is still failing.

**Comment by sarayourfriend** [comment]
Oh dear! It seems our POT file generation is clashing with Windows path delimiters! I created this issue to address that: https://github.com/WordPress/openverse-frontend/issues/1066

For now I'll push the updated file so the check passes for you. Sorry about that!

**Comment by BarokDG** [comment]
> Oh dear! It seems our POT file generation is clashing with Windows path delimiters! I created this issue to address that: #1066
> 
> For now I'll push the updated file so the check passes for you. Sorry about that!

It's alright. This was my first ever contribution. Thanks a lot for being so helpful and kind!

**Review by krysal — APPROVED** [review]
Everything seems correct. Thank you @BarokDG!

**Pull request merged**